web/src/components/nomination/NominateForm.vue

```diff
@@ -224,6 +227,7 @@ export default {
           twitter: '',
           facebook: '',
           website: '',
+          priorPresentation: '',
```

Comment:
We'll need to add this to the payloadFields on line 381 so it gets sent to Airtable. I'll go ahead and create a new column `prior_presentation_url`.

Reply:
@ann-kilzer Thanks I have overlooked that bit.